nodejs12.x/cookiecutter-aws-sam-step-functions-with-connectors/{{cookiecutter.project_name}}/template.yaml

```
@@ -0,0 +1,111 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  {{ cookiecutter.project_name }}
+
+  Sample SAM Template for {{ cookiecutter.project_name }}
+
+Resources:
+  # Serverless Connector                                                 #
+  # More info about Connector Resource: https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-resource-connector.html
+  SfnToStockCheckerFunctionConnector:
+    Type: AWS::Serverless::Connector 
+    Properties:
+      Source:
+        Id: StockTradingStateMachine
+      Destination:
+        Id: StockCheckerFunction
+      Permissions:
+        - Write
```

Consistent with the other resources in the template:

```suggestion
  SfnToStockCheckerFunctionConnector:
    Type: AWS::Serverless::Connector # More info about Connector Resource: https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-resource-connector.html
    Properties:
      Source:
        Id: StockTradingStateMachine
      Destination:
        Id: StockCheckerFunction
      Permissions:
        - Write
```